Packs/PAN-OS/ReleaseNotes/1_17_0.md

```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Palo Alto Networks PAN-OS
+
+- Added support for the ***fetch-incidents*** command to support multiple devices.
+- Improved implementation of the ***fetch-incidents*** command by changing the name of the incident in the Cortex XSOAR machine, from ID to device name and ID.
+- Updated the Docker image to: *demisto/pan-os-python:1.0.0.58193*.
```

Comment:
```suggestion
- Added support for the ***fetch-incidents*** command to support multiple devices.
- Improved implementation of the ***fetch-incidents*** command by changing the name of the incident in the Cortex XSOAR machine, from ID to device name and ID, and receiving the fetched incidents in ascending order instead of descending.  
- Updated the Docker image to: *demisto/pan-os-python:1.0.0.58193*.
```